### ISSUE TITLE
Add protocol Piligrim v-4

### ISSUE THREAD

**Issue by rayder2007** [traccar/traccar#4914]
### Protocol type:
Text

### Examples of messages transmitted by the system:
1. The current status and location of the object (sent in response to the "Get status" command or on a call):
 `&PHONENUMBER=%2B+69999999999&MESSAGE=$GPRMC, 222408.000, V, 6201.4451, N, 03325.5726, E, 1.84, 96.27, 250310 A*55; GSM: 2f0-01, 0f85, 2a90; S; Batt: 381, R`
2. An external power supply has been switched off, indicating the current
coordinates:
`&PHONENUMBER=%2B+69999999999&MESSAGE=POWER RESERVE; $GPRMC, 222408.000, V, 6201.4451, N, 03325.5726, E, 1.84, 96.27, 250310, , A*6B\n; GSM: 2f0-01, 0f85, 2a90; S; Batt: 419, R`
3. An external power supply has been connected, indicating the current
coordinates:
`&PHONENUMBER=%2B+69999999999&MESSAGE=POWER MAIN; $GPRMC, 222408.000, V, 6201.4451, N, 03325.5726, E, 1.84, 96.27, 250310, , A*6B\n; GSM: 2f0-01, 0f85, 2a90; S; Batt: 393, M`
4. The message is sent by the device if the motion sensor is enabled:
`&PHONENUMBER=%2B+69999999999&MESSAGE=Motion detect; $GPRMC, 222408.000, V, 6201.4451, N, 03325.5726, E, 1.84, 96.27, 250310, , A*6B\n; GSM: 2f0-01, 0f85, 2a90; M; Batt: 393, R`
Or
`&PHONENUMBER=%2B+69999999999&MESSAGE=DEPART; $GPRMC, 222408.000, V, 6201.4451, N, 03325.5726, E, 1.84, 96.27, 250310, , A*6B\n; GSM: 2f0-01, 0f85, 2a90; M; Batt: 393, R`
5. Message transmitted by the device when stopped with the "Stop on" command enabled:
`&PHONENUMBER=%2B+69999999999&MESSAGE=Stop detect; $GPRMC, 222408.000, V, 6201.4451, N, 03325.5726, E, 1.84, 96.27, 250310, , A*6B\n; GSM: 2f0-01, 0f85, 2a90; M; Batt: 393, R`
Or
`&PHONENUMBER=%2B+69999999999&MESSAGE=ARRIVE; $GPRMC, 222408.000, V, 6201.4451, N, 03325.5726, E, 1.84, 96.27, 250310, , A*6B\n; GSM: 2f0-01, 0f85, 2a90; M; Batt: 393, R`

### String format $GPRMC (coordinates):
`$GPRMC` prefix
`222408.000` time 22:24:08 (GMT)
`A` A-satellites visible, V-not visible
`6201.4451` 62 degrees 01.74451 minute of latitude
`N` N-north, S-south.
`03325.5726` 33 degrees 25.5726 minutes longitude
`E` E-East, W-West
`6.65` Speed (in knots)
`66.22` Direction (0-server, 180-°)
`250310` date
`GSM:..,..,..;` all visible GSM base stations
`S` motion sensor: S-stand, M-movement
`Batt: 381` battery level (3.81V)
`R` type of power supply: R-redundant, M - external power supply

Examples### 
`&PHONENUMBER=%2B+68922705034&MESSAGE=$GPRMC,204200.000,A,5559.7882,N,09256.7935,E,0.35,341.20,210722,,,A* 28,12; GSM: 250-01 5E5C-1498,1497,0908,0909,FFFF 24;  M; BATT: 361,M`
`&PHONENUMBER=%2B+68916931588&MESSAGE=ARRIVE; $GPRMC,032234.000,A,5637.2227,N,09033.7209,E,0.36,51.89,260722,,,A* 42,21; GSM: 250-01 5E31-06D8,0000,,, 21;  S; BATT: 410,R`

**Comment by tananaev** [comment]
Is this a feature request? We would need protocol documentation.

Also are you interested in sponsoring the work?

**Comment by rayder2007** [comment]
Yes, this is a feature request. This is a simple text protocol. All protocol documentation in the first post =)

**Comment by tananaev** [comment]
OK, but lack of proper documentation lowers the chances that it will be implemented.

**Comment by tananaev** [comment]
Do you have logs from the device?
